### PR TITLE
Support for MusicBrainz Picard

### PR DESCRIPTION
**Pull request by TrigonaMinima**
https://musicbrainz.org/

User configuration file in ```.config/MusicBrainz/```

### CODE REVIEW

File: mackup/applications/musicbrainz-picard.cfg [hunk 1]
```diff
@@ -0,0 +1,5 @@
+[application]
+name = MusicBrainz Picard
+
+[configuration_files]
+.config/MusicBrainz
```
Comment (vitorgalvao) [inline mackup/applications/musicbrainz-picard.cfg]: It has a “no newline at end of file” warning. Those are [actually meaningful](http://unix.stackexchange.com/a/18789).

File: mackup/applications/picard.cfg [hunk 2]
```diff
@@ -0,0 +1,5 @@
+[application]
+name = MusicBrainz Picard
+
+[configuration_files]
+.config/MusicBrainz
```
Comment (vitorgalvao) [inline mackup/applications/picard.cfg]: It has a “no newline at end of file” warning. Those are [actually meaningful](http://unix.stackexchange.com/a/18789).

Comment (TrigonaMinima) [inline mackup/applications/picard.cfg]: Thanks for the link.

**Comment by TrigonaMinima** [comment]
You can merge this.

**Comment by lra** [comment]
how?
screen shot 2015-06-08 at 1 11 49 pm

**Comment by lra** [comment]
Also there are still missing newlines.

**Comment by lra** [comment]
and 2 config files are about the same application.

**Comment by TrigonaMinima** [comment]
Now, this should merge.

**Pull request merged**